### ISSUE TITLE
Training the image branch

### ISSUE THREAD

**Issue by Sharpiless** [haoy945/DeMF#5]
I trained the deformable detr and got:
```
+-------------+-------+--------+--------+-------+
| class       | gts   | dets   | recall | ap    |
+-------------+-------+--------+--------+-------+
| bed         | 515   | 37988  | 0.911  | 0.337 |
| table       | 2348  | 108875 | 0.852  | 0.228 |
| sofa        | 627   | 35760  | 0.856  | 0.186 |
| chair       | 10016 | 196779 | 0.795  | 0.297 |
| toilet      | 145   | 11759  | 0.924  | 0.330 |
| desk        | 1882  | 57401  | 0.633  | 0.079 |
| dresser     | 218   | 15207  | 0.826  | 0.143 |
| night_stand | 255   | 14305  | 0.773  | 0.277 |
| bookshelf   | 282   | 18454  | 0.709  | 0.113 |
| bathtub     | 49    | 8472   | 0.653  | 0.066 |
+-------------+-------+--------+--------+-------+
| mAP         |       |        |        | 0.205 |
+-------------+-------+--------+--------+-------+
```

with 2 gpus and ÷4 lr config runing:
```
CUDA_VISIBLE_DEVICES=4,5 bash tools/dist_train.sh configs/deformdetr/imvotenet_deform.py 2
```

Only the learning rate has been modified:

```
# optimizer
optimizer = dict(
    type='AdamW',
    lr=2e-4 / 4,
    weight_decay=0.0001,
    paramwise_cfg=dict(
        custom_keys={
            'backbone': dict(lr_mult=0.1, decay_mult=1.0),
            'sampling_offsets': dict(lr_mult=0.1, decay_mult=1.0),
            'reference_points': dict(lr_mult=0.1, decay_mult=1.0)
        })
    )
```

Is this normal? Any suggestions would be greatly appreciated.

logs available at [https://drive.google.com/file/d/1tevboRqN-U0orZn-Jw_Pw_4OahkDcuP4/view?usp=sharing](https://drive.google.com/file/d/1tevboRqN-U0orZn-Jw_Pw_4OahkDcuP4/view?usp=sharing)

**Comment by Sharpiless** [comment]
logs available at [https://drive.google.com/file/d/1tevboRqN-U0orZn-Jw_Pw_4OahkDcuP4/view?usp=sharing](https://drive.google.com/file/d/1tevboRqN-U0orZn-Jw_Pw_4OahkDcuP4/view?usp=sharing)

**Comment by chenshi3** [comment]
It is not normal. It should be as same the faster-rcnn version.

**Comment by chenshi3** [comment]
And you can try to train the image branch in Imvotenet, they should have similar performance.

**Comment by Nihilianth** [comment]
I was having the same issue. Fixed by using pre-trained Resnet. @chenshi3 is that intended or we're doing something wrong?😅
